Golem.Tests/LogFilesTest.cs

```diff
@@ -0,0 +1,146 @@
+using System.Text.RegularExpressions;
+using System.Threading.Channels;
+
+using Golem.Tools;
+using Golem.Yagna.Types;
+
+using GolemLib;
+using GolemLib.Types;
+
+using Microsoft.Extensions.Logging;
+
+namespace Golem.Tests
+{
+    [Collection(nameof(SerialTestCollection))]
+    public class LogFilesTest : JobsTestBase
+    {
+
+        public LogFilesTest(ITestOutputHelper outputHelper, GolemFixture golemFixture)
+            : base(outputHelper, golemFixture, nameof(LogFilesTest))
+        { }
+
+        [Fact]
+        public async Task StartRunStopStartStopStartStop_VerifyLogFiles()
```

Comment:
This test seems like an overkill, but the `LogFiles` property will be ignored until there will be some error to diagnose, so I thought it would be better to have a test which will fail if e.g. `ya-provider` will change its `exe-unit` dir name.